docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,634 @@
+
+1. Declare a frame-component in the manifest file:
+    ```
+    <component-based-application appid="org.tizen.base-component" exec="@BINDIR@/base-component" nodisplay="false" multiple="false" type="capp">
+        <label>Base-component application</label>
+        <icon>@DESKTOP_ICON@</icon>
+        <frame-component id="base-frame" launch_mode="caller" main="true" icon-display="false" taskmanage="true">
+            <icon>org.tizen.sample.png</icon>
+            <label>base-frame</label>
+            <label xml:lang="en-us">base-frame</label>
+            <label xml:lang="ko-kr">base-frame[KOR]</label>
+        </frame-component>
+    </component-based-application>
+    ```
+
+2. Add a frame-component to the component-based application:
+    ```
+    component_class_h frame_component_add(component_class_h comp_class,
+            const char *component_id, void *user_data)
+    {
+        frame_component_lifecycle_callback_s callback = {
+            .create = __frame_component_create_cb,
+            .start = __frame_component_start_cb,
+            .resume = __frame_component_resume_cb,
+            .pause = __frame_component_pause_cb,
+            .stop = __frame_component_stop_cb,
+            .destroy = __frame_component_destroy_cb,
+            .restore_content = __frame_component_restore_content_cb,
+            .save_content = __frame_component_save_content_cb,
+            .action = __frame_component_action_cb,
+            .device_orientation_changed = __frame_component_device_orientation_changed_cb,
+            .language_changed = __frame_component_language_changed_cb,
+            .region_format_changed = __frame_component_region_format_changed_cb,
+            .low_battery = __frame_component_low_battery_cb,
+            .low_memory = __frame_component_low_memory_cb,
+            .suspended_state_changed = __frame_component_suspended_state_changed_cb,
+        };
+
+        return component_based_app_add_frame_component(comp_class,
+                component_id, &callback, user_data);
+    }
+
+    component_class_h __app_create_cb(void *user_data)
+    {
+         component_class_h comp_class = NULL;
+         comp_class = frame_component_add(comp_class, "base-frame", NULL);
+        return comp_class;
+    }
+
+    void __app_terminate_cb(void *user_data)
+    {
+    }
+
+    int main(int argc, char** argv)
+    {
+        int ret;
+        component_based_app_lifecycle_callback_s callback = {
+            .create = __app_create_cb,
+            .terminate = __app_terminate_cb
+        };
+
+        ret = component_based_app_main(argc, argv, &callback, NULL);
+        if (ret != APP_ERROR_NONE)
+            return ret;
+
+        return 0;
+    }
+    ```
+
+<a name="service_component_monitoring"></a>
+## Managing Service-component
+To add and manage service-component, you must add service-component and register the service-component callbacks.
```

Comment:
Please add "a" before "service-component".
...you must add a service-component and register the service-component callbacks.